### PR TITLE
Some rules for  Issue 575

### PR DESCRIPTION
**Pull request by frack113**
done
- 42 | None | FromBase64String / Compression / X509Enrollment
- 43 | None | FromBase64String + Compression

### CODE REVIEW

File: rules/windows/powershell/powershell_script/posh_ps_x509enrollment.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+date: 2022/12/23
+logsource:
+    product: windows
+    category: ps_script
+    definition: Script block logging must be enabled
+detection:
+    selection:
+        ScriptBlockText|contains:
+            - 'X509Enrollment.CBinaryConverter'
+            - '884e2002-217d-11da-b2a4-000e7bbb2b09'
```
Comment (nasbench) [inline rules/windows/powershell/powershell_script/posh_ps_x509enrollment.yml:21]: Can't find malicious scripts on VT using this and the screenshot didn't show any result for this. Can you find examples?

File: rules/windows/process_creation/proc_creation_win_frombase64string_archive.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+title: Suspicious FromBase64String Gzip Archive - Process Creation
+id: d75d6b6b-adb9-48f7-824b-ac2e786efe1f
+related:
+    - id: df69cb1d-b891-4cd9-90c7-d617d90100ce
+      type: similar
+status: experimental
+description: Detect use of a Gzip archive in a base64 string
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_frombase64string_archive.yml:7]:
```suggestion
description: Detects attempts of decoding a base64 Gzip archive via PowerShell. This technique is often used as a method to load malicious content into memory afterward.
```

File: rules/windows/process_creation/proc_creation_win_frombase64string_archive.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+title: Suspicious FromBase64String Gzip Archive - Process Creation
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_frombase64string_archive.yml:1]:
```suggestion
title: Suspicious FromBase64String Usage On Gzip Archive - Process Creation
```

File: rules/windows/powershell/powershell_script/posh_ps_frombase64string_archive.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+title: Suspicious FromBase64String Gzip Archive - Ps Script
```
Comment (nasbench) [inline rules/windows/powershell/powershell_script/posh_ps_frombase64string_archive.yml:1]:
```suggestion
title: Suspicious FromBase64String Usage On Gzip Archive - Ps Script
```

File: rules/windows/process_creation/proc_creation_win_frombase64string_archive.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+author: frack113
+date: 2022/12/23
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection:
+        CommandLine|contains|all:
+            - 'FromBase64String'
+            - 'H4sI'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_frombase64string_archive.yml:19]: Same as above

```suggestion
            - 'FromBase64String'
            - 'MemoryStream'
            - 'H4sI'
```

File: rules/windows/process_creation/proc_creation_win_x509enrollment.yml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+author: frack113
+date: 2022/12/23
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection:
+        CommandLine|contains:
+            - 'X509Enrollment.CBinaryConverter'
+            - '884e2002-217d-11da-b2a4-000e7bbb2b09'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_x509enrollment.yml:20]: Same as comment above

File: rules/windows/powershell/powershell_script/posh_ps_x509enrollment.yml (hunk 1 above)
Comment (frack113) [reply rules/windows/powershell/powershell_script/posh_ps_x509enrollment.yml:21]: https://hybrid-analysis.com/sample/3db05b03b7384ec90a16fdb0a23292aa7aa1090d795802c58a2beb844fc5d763/6341f9c40d08c7439445ff13
Not sure can be set to low

Comment (nasbench) [reply rules/windows/powershell/powershell_script/posh_ps_x509enrollment.yml:21]: Here it is https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse?slide=41

We can keep it as a medium rule and include this in the reference

**Review by nasbench — APPROVED** [review]

**Pull request merged**